js_modules/dagster-ui/packages/ui-core/src/assets/AssetHealthSummary.tsx

```diff
@@ -236,6 +246,16 @@ const Criteria = React.memo(
               </Link>
             </Body>
           );
+        case 'AssetHealthFreshnessMeta':
+          if (metadata.lastMaterializedTimestamp === null) {
+            return <Body>No materializations</Body>;
+          }
+
+          return (
+            <Body>
+              Last materialized {dayjs(Number(metadata.lastMaterializedTimestamp)).fromNow()} ago
```

Comment:
@salazarm i don't know why, but this is giving me output like `55 days ago` (see the screenshots in the description) - i checked the GQL query response and the timestamp is correct (example 1744821283.3374982) so i must be doing some incorrect conversion here?